_adr/87/index.adoc

```diff
@@ -1,48 +1,452 @@
+** This format is used for all of our recording rules, most are not exposed to end users.
+** This makes sense when you know how it's implemented but this exposes implementation details to end users.
+** A list of the customer facing recording rules can be seen in the https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/blob/main/resources/prometheus/user-facing-recording-rules.yaml[observability-resources-mk] configuration repository.
+
+
+### Exposing new metrics in the KAS Fleet Manager API
+All of the customer facing metrics originate from the data plane cluster in which the Kafka instance is deployed in.
+A Prometheus instance is deployed in each of the data plane cluster which uses https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write[remote-write] to push these metrics to Observatorium.
+This acts as the central repository for all metrics, included in the remote-write configuration, from each data plane cluster.
+A list of the metrics included in the remote-write configuration can be found in the https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/blob/main/resources/prometheus/remote-write.yaml[Observability resources] repository.
+
+KAS Fleet Manager acts as a proxy to Observatorium.
+It calls the Observatorium _/query_ and _/query_range_ endpoints to retrieve metrics we consider as 'customer facing'.
+A list of the metrics retrieved by KAS Fleet Manager can be found in the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/pkg/client/observatorium/api.go[observatorium] client within the project. When new metrics are added to this list, they become available in the KAS Fleet Manager _/metrics/query_ and _/metrics/query_range_ endpoints. In order to make new metrics available to the _/metrics/federate_ endpoint, their metadata must be added to this https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/constants/metrics.go[file] in KAS Fleet Manager.
+
+KAS Fleet Manager controls the labels exposed for each metric returned in all endpoints. 
+Metrics retrieved from Observatorium are also filtered so that end users will only receive metrics for their own Kafka instance.
+Apart from these, no additional aggregation, filtering or modification of the metric is done by KAS Fleet Manager.
 
 ## Proposal
-// What is the decision being proposed
+### Metric and Label Naming
+All metric and label names must adhere to the policy stated in link:../../_ap/16/index.adoc[AP-16: Naming Policy for Customer Facing Metrics].
+
+### Kafka Metric Categories
+The customer facing Kafka metrics can be divided into multiple categories.
+Categorizing these metrics into logical groups can help us formulate the metric name and document it.
+
+|===
+|Category | Description | Sub-component (to be specified in the metric name)
+
+|Cluster
+|General Kafka metrics not specific to any other sub-components.
+|cluster
```

Comment:
I would suggest that we should be using the terminology `instance` rather than `cluster`.  The fact that we are giving each instance its own cluster is really an implementation detail.